mongo/options/clientoptions.go

```diff
@@ -760,6 +765,13 @@ func (c *ClientOptions) SetServerAPIOptions(opts *ServerAPIOptions) *ClientOptio
 	return c
 }
 
+// SetSRVServiceName specifies a custom SRV service name to use in SRV polling. To use a custom SRV service name
+// in SRV discovery, this function must be called before ApplyURI.
```

Comment:
I suggest adding a note like: `This can also be set through the "srvServiceName" URI option.` for consistency with other options.

Reply:
Done.